pages/developers/packaging.md

```diff
@@ -196,6 +196,27 @@ run, that version will be forced, otherwise, it works as normal.
 > Python one](https://github.com/github/gitignore/blob/main/Python.gitignore)
 > or using a [generator site](https://www.toptal.com/developers/gitignore).
 
+You should also add these two files:
+
+`.git_archival.txt`:
+
+```text
+node: $Format:%H$
+node-date: $Format:%cI$
+describe-name: $Format:%(describe)$
```

Comment:
```suggestion
describe-name: $Format:%(describe:tags=true,match=*[0-9]*)$
```

@jakenmen, is this a better recommendation, as mentioned on discord?

Reply:
Yes.

Longer answer:

`:tags=true` makes it so that it works with unannotated tags (which you probably do because GitHub Releases uses those and `git tag` will only create annotated tag if you pass specific options to it). This is definitely better and unless you have a very specific workflow, the functionality won't even work at all without `:tags=true`. The only problem is that *right now*, GitHub doesn't use a new enough Git version to support it but that surely will get resolved in a few months and it's not like *not* using it will give you any better results when you're not using annotated tags.

`,match=*[0-9]*` makes it only match tags with at least one number in them, this is the default used by `setuptools_scm` and it doesn't seem like you use an option that would change that behavior so this should definitely be used to keep the behavior consistent.